src/desktop/apps/consign/components/choose_artist/__tests__/ChooseArtist.jest.tsx

```diff
@@ -0,0 +1,49 @@
+import ChooseArtist from "desktop/apps/consign/components/choose_artist"
+import * as actions from "desktop/apps/consign/client//actions"
+import React from "react"
+import reducers from "desktop/apps/consign/client/reducers"
+import { createStore } from "redux"
+import { mount } from "enzyme"
+
+describe("ChooseArtist", () => {
+  let initialStore
+
+  beforeEach(() => {
+    initialStore = createStore(reducers)
+  })
+
+  it("disables the button when there is no input", () => {
+    const wrapper = mount(<ChooseArtist store={initialStore} />)
+    expect(
+      wrapper
+        .find(".consignments-submission-choose-artist__next-button[disabled]")
+        .props().disabled
+    ).toBe(true)
```

Comment:
```suggestion
    expect(
      wrapper
        .find(".consignments-submission-choose-artist__next-button[disabled]")
        .length
    ).toEqual(1)
```

The codemod translated these assertions a bit funny, I think. Line 19 says "give me the button that is disabled", and line 20/21 say "make sure the button that is disabled is disabled". 

I am not totally certain on the syntax of my suggestion, but I think it'd be worth updating these to assertions that don't unnecessarily inspect the `props` bag.

Reply:
This wasn't a codemod but an issue with Enzyme and perhaps how i swapped `shallow` for `render`: https://github.com/enzymejs/enzyme/issues/336. It's pretty minor and the overall behavior is still being checked. (This applies to all the following tests.)